README.md

```diff
@@ -24,41 +24,68 @@ The [BrainFlow](https://BrainFlow.org) library provides a uniform API that is de
 7. Turn on your headband
 8. Run the script `main.py` with your device id. The command for running with a [Muse 2 headband](https://choosemuse.com/muse-2/) would be: `python .\main.py --board-id 38`
 
-**OSC Avatar Parameters**
-
-Avatar parameters being sent are floats that range from -1.0 to 1.0. Negative and Positive values correspond to low and high focus/relaxation. Update your avatar paramaters as needed to influnece animations and the like. Parameters are also seperated by left and right sides of the brain. Have fun!
-
-- `osc_focus_left`
-- `osc_relax_left`
-- `osc_focus_right`
-- `osc_relax_right`
-- `osc_focus_avg`
-- `osc_relax_avg`
-
-For easier startup, I've added a hue shift parameter based on the focus and relax values. This parameter will range from 0.0 to 1.0.
-
-- `HueShift`
-
-Added are optional paramaters that appear based on whether or not your headband supports it.
-- `osc_battery_lvl` (int [0-100])
-- `osc_heart_bpm` (int[0-255])
-- `osc_oxygen_percent` (float[0.0-1.0])
-- `osc_respiration_bpm` (int[0-255])
-
-I've also added the alpha, beta, theta, delta, and gamma band power ratios between left, right sides of the head as well as avg. You can access them via this path:
-- `osc_band_power_(head_location)_(brainwave name)` (float [0-1]) (without the parenthesis)
-
-For debug, a boolean paramater is sent to monitor the connection status of the headband as well as a time difference parameter to show the difference between last sample time and current time in seconds.
-
-- `osc_is_connected` (boolean)
-- `osc_time_diff` (float)
+**OSC Avatar Parameter Schema**
+
+Here are the various avatar parameters sent to VRChat. Focus and relax scores range from -1 to 1, corresponding to much focus and relaxation you're experiencing. Brain wave power bands are also sent as floats from 0 to 1. Depending on the board you're using, respiration data might be available.
+
+```yaml
+brainflow:
+  neurofeedback:
+    left:
+      - focus [float]
+      - relax [float]
+    right:
+      - focus [float]
+      - relax [float]
+    avg:
+      - focus [float]
+      - relax [float]
+  power_ratios:
+    left:
+      - alpha [float]
+      - beta [float]
+      - theta [float]
+      - delta [float]
+      - gamma [float]
+    right:
+      - alpha [float]
+      - beta [float]
+      - theta [float]
+      - delta [float]
+      - gamma [float]
+    avg:
+      - alpha [float]
+      - beta [float]
+      - theta [float]
+      - delta [float]
+      - gamma [float]
+  addons:
+    - Hueshift [float 0-1]
+  respiration: # board depedent
+    - oxygen_percent [float]
+    - heart_freq [float]
+    - heart_bpm [int]
+    - respiration_freq [float
+    - respiration_bpm [int]
+  device:
+    - time_diff [float]
+    - is_connected [bool]
+    - battery_lvl [float] # board depedent

```

Comment:
Typo in "dependent", also if this is considered a device-specific feature, perhaps we may want to figure out a better way to represent this so we can tell if this feature is present or not from the unity side

Reply:
Will approach as a separate issue